pkg/bootkube/create.go

```diff
@@ -44,6 +46,9 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 	if err := wait.Poll(5*time.Second, timeout, createFn); err != nil {
 		return fmt.Errorf("Failed to create assets: %v", err)
 	}
+
+	fmt.Println("Spawning self-hosted control plane...")
```

Comment:
Regardless of adding messages to indicate when cluster assets are submitted and messages indicating that this is all complete, I still think this message is useful because it indicates what is happening with the indented pod status messages that follow.

Playing around with exposing asset creation messages as well.